src/styles/card.scss

```diff
@@ -0,0 +1,137 @@
+html{
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+}
+
+.main {
+  margin: 50px 40px;
+  padding: 0;
+  display: grid;
+  justify-content: center;
+  row-gap: 48px;
+  column-gap: 46px;
+  grid-template-rows: repeat(8, 408px);
+  grid-template-columns: repeat(1, 200px);
+  @media (min-width: 488px) {
+    grid-template-rows: repeat(4, 408px);
+    grid-template-columns: repeat(2, 200px);
+  }
+  @media (min-width: 768px) {
+    grid-template-rows: repeat(3, 408px);
+    grid-template-columns: repeat(3, 200px);
+  }
+  @media (min-width: 1024px) {
+    grid-template-rows: repeat(2, 408px);
+    grid-template-columns: repeat(4, 200px);
+  }
+}
+
+
+
+.product-card {
+  background-color: $main-page-color;
+  box-sizing: border-box;
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid rgba(243, 243, 243, 1);
+  transition: all 0.3s;
+
+  &:hover {
+    transform: scale(1.2);
+  }
+
+  &__image {
+    background-repeat: no-repeat;
+    background-position: center top;
+    margin-top: 32px;
+    background-image: url("../images/imac.jpeg");
+    background-size: 160px 134px;
+    height: 134px;
+    width: 100%;
+  }
+
+  &__content {
+    margin: 0 16px;
+
+    &:hover .card__title{
+      color: $hover-text-color;
+    }
+  }
+}
+
+.card__title {
+  width: 166px;
+  font-size: 12px;
+  margin: 0;
+  padding-block: 40px 0;
+  line-height: 18px;
+  color: $text-color;
+  font-weight: 500;
+  transition: all 0.3s;
+
+}
+
+.text__code {
+  margin: 0;
+  padding-top: 4px;
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary-text-color;
+}
+
+.content__mark {
+  padding-top: 16px;
+  height: 16px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.mark__reviews {
+  color: $text-color;
+  font-size: 10px;
+  line-height: 14px;
+}
+
+
+.content__price {
+  display: flex;
+  justify-content: space-between;
+  padding-top: 24px;
+}
+.content__price {
+  color: $secondary-text-color;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+
+.price__value {
+  color: $text-color;
+  font-weight: 700;
+  font-size: 16px;
+  line-height: 18px;
+}
+
+.buy__button {
+  box-sizing: border-box;
+  padding: 0;
+  border: none;
+  margin-top: 16px;
+  width: 166px;
+  height: 40px;
+  border-radius: 5px;
+  background-color: $button-color;
+  color: $main-page-color;
+  font-size: 14px;
+  font-weight: 700;
+  line-height: 16px;
+  transition: all 0.3s;
+
+  &:hover {
+    background-color: #fff;
+    border: 1px solid $button-color;
+    color: #00acdc;
```

Comment:
The color `#00acdc` is hardcoded. Consider using a variable for this color to maintain consistency and make future changes easier.